javascript/ql/src/semmle/javascript/frameworks/ClientRequests.qll

```diff
@@ -247,12 +242,10 @@ module ClientRequest {
       method = "request" and
       result = getOptionArgument(0, "data")
       or
-      (method = "post" or method = "put" or method = "put") and
-      (result = getArgument(1) or result = getOptionArgument(2, "data"))
+      method = ["post", "put", "put"] and
```

Comment:
Huh, `put` was mentioned twice, but I think we might as well remove the second mention now that we're at it.